package.json

```diff
@@ -17,6 +17,7 @@
         "grunt-contrib-watch": "~1.0.0",
         "grunt-crx": "^1.0.5",
         "grunt-replace-json": "^0.1.0",
+        "grunt-webext-builder": "^0.1.0",
```

Comment:
Please remove `web-ext` from optional dependencies since we've not using it directly now.